pkg/kubectl/cmd/certificates.go

```diff
@@ -225,7 +225,7 @@ func (options *CertificateOptions) modifyCertificateCondition(builder *resource.
 			csr := info.Object.(*certificatesv1beta1.CertificateSigningRequest)
 			csr, hasCondition := modify(csr)
 			if !hasCondition || force {
-				csr, err = clientSet.Certificates().
+				csr, err = clientSet.CertificatesV1beta1().
```

Comment:
Since you're touching it, I'd rather you switch the client to something similar to here: https://github.com/kubernetes/kubernetes/blob/0e31372b2fa6dc9e6a32f4be09c088ec0adfdf8d/pkg/kubectl/cmd/create/create_job.go#L119